extensions/interactions/LogicProof/LogicProof.js

```diff
@@ -47,24 +47,45 @@ oppia.directive('oppiaInteractiveLogicProof', [
           logicProofData.BASE_STUDENT_LANGUAGE,
           ['variable', 'constant', 'prefix_function']
         );
-        $scope.questionData.language.operators = $scope.typing[0].operators;
 
-        $scope.assumptionsString = logicProofShared.displayExpressionArray(
-          $scope.questionData.assumptions,
-          $scope.questionData.language.operators);
+        $scope.operators = $scope.typing[0].operators;
+
+
+        $scope.displayExpression = function(expression) {
+          return logicProofShared.displayExpression(
+            expression, $scope.questionData.language.operators);
+        };
+
+        $scope.displayExpressionArray = function(array) {
```

Comment:
I thought you were either going to add a comment to these, or remove the separate methods?